lib/yabeda/active_record.rb

```diff
@@ -71,7 +71,12 @@ class Error < StandardError; end
 
       # Connection pool metrics collection
       collect do
-        connection_pools = ::ActiveRecord::Base.connection_handler.connection_pool_list
+        connection_pools = \
+          if ::ActiveRecord.version >= "7.1"
```

Comment:
```suggestion
          if ::ActiveRecord.version >= Gem::Version.new("7.1")
```

Now CI revealed that older Ruby versions couldn't do `Gem::Version` comparison with strings (it [was implemented only in 3.2](https://github.com/ruby/ruby/commit/9828502570acce53f4094c23756bc82bd256eab7)), so need to do it explicitly.

Reply:
Whops 😅